test/unit/specs/components/WwwFrame/SearchBar.spec.js

```diff
@@ -32,17 +32,17 @@ describe('SearchBar', () => {
 		});
 	});
 
-	it('should fetch suggestions when it gains focus', () => {
+	it('should fetch suggestions when it gains focus', done => {
 		const input = wrapper.find({ ref: 'input' });
 		input.trigger('focus');
-		input.trigger('blur');
-		wrapper.vm.focus();
```

Comment:
@emuvente I think this may have been meant to give focus to the input element when clicking anywhere in the form...but maybe not. Do you see any issue with just checking the standard focus within the input?

Reply:
This had originally been 
```
input.trigger('focus');
input.trigger('blur');
input.trigger('focus');
```
This was to test that each time the search bar was focused on it would fetch suggestions. Cypress wouldn't properly focus the second time doing that, so I changed the second focus command to just call the focus method directly using `wrapper.vm.focus()`.

I think it's probably fine not to test for that anymore, as it's not that important.

Reply:
Fixed this test back to its original intent of focusing twice

Reply:
@eddieferrer did cypress pass ok with this in place?

Reply:
If not we could just remove or possibly give explicit focus to something else first, such as the form button then focus back into the input.